src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -14,72 +14,78 @@
+    /** during assembly, don't explicitly merge an overlap if shorter than this value */
+    public int getMinMergeLen() { checkOpen(); return opts.getInt(20); }
+    public void setMinMergeLen( final int minMergeLen ) { checkOpen(); opts.putInt(20,minMergeLen); }
+
+    /** graph cleaning options -- you'll have to look at the fermi lite code to try to figure out what these do */
+    public int getCleaningFlag() { checkOpen(); return opts.getInt(24); }
+    public void setCleaningFlag( final int cleaningFlag ) { checkOpen(); opts.putInt(24,cleaningFlag); }
+    public int getCleaningMinOverlap() { checkOpen(); return opts.getInt(28); }
+    public void setCleaningMinOverlap( final int minOverlap ) { checkOpen(); opts.putInt(28,minOverlap); }
+    public int getCleaningELen() { checkOpen(); return opts.getInt(32); }
+    public void setCleaningELen( final int eLen ) { checkOpen(); opts.putInt(32,eLen); }
+    public int getCleaningMinEnsr() { checkOpen(); return opts.getInt(36); }
+    public void setCleaningMinEnsr( final int minEnsr ) { checkOpen(); opts.putInt(36,minEnsr); }
+    public int getCleaningMinInsr() { checkOpen(); return opts.getInt(40); }
+    public void setCleaningMinInsr( final int minInsr ) { checkOpen(); opts.putInt(40,minInsr); }
+    public int getCleaningMaxBDist() { checkOpen(); return opts.getInt(44); }
+    public void setCleaningMaxBDist( final int maxBDist ) { checkOpen(); opts.putInt(44,maxBDist); }
+    public int getCleaningMaxBDiff() { checkOpen(); return opts.getInt(48); }
+    public void setCleaningMaxBDiff( final int maxBDiff ) { checkOpen(); opts.putInt(48,maxBDiff); }
+    public int getCleaningMaxBVtx() { checkOpen(); return opts.getInt(52); }
+    public void setCleaningMaxBVtx( final int maxBVtx ) { checkOpen(); opts.putInt(52,maxBVtx); }
+    public int getCleaningMinMergeLen() { checkOpen(); return opts.getInt(56); }
+    public void setCleaningMinMergeLen( final int minMergeLen ) { checkOpen(); opts.putInt(56,minMergeLen); }
+    public int getCleaningTrimLen() { checkOpen(); return opts.getInt(60); }
+    public void setCleaningTrimLen( final int trimLen ) { checkOpen(); opts.putInt(60,trimLen); }
+    public int getCleaningTrimDepth() { checkOpen(); return opts.getInt(64); }
+    public void setCleaningTrimDepth( final int trimDepth ) { checkOpen(); opts.putInt(64,trimDepth); }
+    public float getCleaningDRatio1() { checkOpen(); return opts.getFloat(68); }
+    public void setCleaningDRatio1( final float dRatio1 ) { checkOpen(); opts.putFloat(68,dRatio1); }
+    public float getCleaningMaxBCov() { checkOpen(); return opts.getFloat(72); }
+    public void setCleaningMaxBCov( final float maxBCov ) { checkOpen(); opts.putFloat(72,maxBCov); }
+    public float getCleaningMaxBFrac() { checkOpen(); return opts.getFloat(76); }
+    public void setCleaningMaxBFrac( final float maxBFrac ) { checkOpen(); opts.putFloat(76,maxBFrac); }
+    public int getExpectedOptsSize() { return 80; }
```

Comment:
Lets make these package scoped because I don't there's any reason to expose them to non-test clients.